README.md

```diff
@@ -513,15 +496,7 @@ the `Todo` key and render it.
 
 ## Rendering a List
 
-**✏️ Modify the contents of** `frontend/main.tsx`**:**
-
-- Get rid of the default Vite styles by removing line 4:
-  `import './index.css'`
-
-- Remove the extension from line 3, it should be:
-  `import App from './App'`
-
-`main.tsx` should look like:
```

Comment:
We should either give step-by-step instructions, or show the final result, but probably not both (I found it confusing)

Reply:
i think we need the step-by-step so it's clear _what_ we're doing, the final code is a request from Justin as it's always nice to be able to just copy&paste the working bit in, ideally if this would be in the format like the bitovi academy trainings where you can have the collapsed code and only highlight the line changes